natives.json

```diff
@@ -57678,31 +57678,31 @@
 			"comment": "",
 			"params": [
 				{
-					"type": "Any*",
+					"type": "const char*",
 					"name": "p0"
 				},
 				{
-					"type": "Any",
+					"type": "int",
 					"name": "p1"
 				},
 				{
-					"type": "Any",
+					"type": "int",
 					"name": "p2"
 				},
 				{
-					"type": "Any",
+					"type": "int",
 					"name": "p3"
 				},
 				{
-					"type": "Any*",
+					"type": "const char*",
 					"name": "p4"
 				},
 				{
 					"type": "BOOL",
 					"name": "p5"
 				}
 			],
-			"return_type": "Any",
+			"return_type": "Any*",

```

Comment:
This is actually an integer (a hash I think).

Reply:
Can you maybe give me some insight on why this is actually an integer?
The decompiled source always uses the native like this:
```c
´*uParam0 = NETWORK::UGC_TEXTURE_DOWNLOAD_REQUEST(sParam1, iParam4, iParam2, 0, sParam1, false);
uParam0->f_1 = 2;
```
So from my understanding it has to be a struct.

Reply:
The decompiler doesn't have the nicest output for struct fields at offset `0`.
That `*uParam0 = ...` here is actually `*uParam0.f_0 = ...`, which is an integer (and seems to be a hash from what I can see in game code).

Reply:
Ah okay, makes sense then.